sdk/nodejs/Makefile

```diff
@@ -112,6 +112,18 @@ ifneq ($(PULUMI_TEST_COVERAGE_PATH),)
 	if [ -e .nyc_output ]; then yarn run nyc report -r cobertura --report-dir $(PULUMI_TEST_COVERAGE_PATH); fi
 endif
 
+PULUMI_TEST_ORG ?= $(shell pulumi whoami --json | jq ".organizations[0]")
+TEST_GREP ?= ".*"
+
+# Watches all the typescript files and runs the tests every time they change.
+# Can optionally be given a regex of tests to match. For example, to iterate
+# on one test specifically, one could use the following command:
+#
+# $ make test_watch TEST_GREP="lists tag values"
+test_watch::
+	echo $(PULUMI_TEST_ORG)
+	PULUMI_TEST_ORG=$(PULUMI_TEST_ORG) yarn mocha "**/*.spec.ts" --timeout 300000 --bail -j 1 --watch --watch-files "**/*.ts" --grep "$(TEST_GREP)"
```

Comment:
Neat!  I like the `TEST_GREP` pattern, I think it would be lovely to add that to all `make test*` variants if possible.  I always forget which incantation I need to use to run a specific test. (This doesn't need to happen in this PR of course)